zebrad/src/components/mempool.rs

```diff
@@ -438,6 +454,8 @@ impl Service<Request> for Mempool {
             }
```

Comment:
The test failure is from [this line above](https://github.com/ZcashFoundation/zebra/pull/6154/files#diff-ec634b649fdc36d20e3746cb9106eefafff1315ce871dd9b16fbbe5708d85784R453):
```rust
self.transaction_sender.send(send_to_peers_ids)?;
```

Reply:
Fixed in [`4b85ddc`](https://github.com/ZcashFoundation/zebra/pull/6154/commits/4b85ddc853d06559aa6f7a4438c8768ee249b94e).